packages/dds/register-collection/src/consensusRegisterCollection.ts

```diff
@@ -213,8 +214,9 @@ export class ConsensusRegisterCollection<T>
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.loadCore}
      */
     protected async loadCore(storage: IChannelStorageService): Promise<void> {
-        const header = await storage.read(snapshotFileName);
-        const dataObj = header !== undefined ? this.parse(fromBase64ToUtf8(header), this.serializer) : {};
+        const blob = await storage.readBlob(snapshotFileName);
+        const header = bufferToString(blob);
+        const dataObj = this.parse(header, this.serializer);
```

Comment:
same here - no need to have this.parse as a function - there is already higher level abstraction of reading/parsing in the form of deserializeBlob